Libraries/ART/ARTSurfaceView.m

```diff
@@ -47,6 +47,7 @@ - (void)invalidate
 
 - (void)drawRect:(CGRect)rect
 {
+  [super drawRect:rect];
```

Comment:
Is this part necessary for this change set?

Reply:
As I recall, this and the `[super drawRect:]` call in RCTTextView.m are necessary to respond to system theme changes.

Just FYI most of the iOS implementation in the PR is was originally developed in the MS fork of react-native back in August of last year (https://github.com/microsoft/react-native/pull/105) and has been shipping in Microsoft products since then.